NEWS.md

```diff
@@ -4,11 +4,15 @@
 * `vec_slice()` and `vec_chop()` now work correctly with `bit64::integer64()`
   objects when an `NA` subscript is supplied. By extension, this means that
   `vec_init()` now works with these objects as well (#813).
-  
+
 * `vec_rbind()` now binds row names. When named inputs are supplied
   and `names_to` is `NULL`, the names define row names. If `names_to`
   is supplied, they are assigned in the column name as before.
 
+* `vec_cbind()` now binds row names if they are congruent across
+  inputs. If the row names are not identical that's an error. If some
+  inputs do not have row names, they are propagated.
```

Comment:
What does propagated mean in this context?

Reply:
That they are assigned the existing rownames. I have removed this sentence as it doesn't seem that informative.